workflow/airflow_libero/README.md

```diff
@@ -0,0 +1,116 @@
+# Airflow Example Usage
+
+Dependencies
+------------
+
+* Python >=3.6
+* [pipenv](https://docs.pipenv.org/)
+
+Installation
+------------
+This will setup your virtual environment and install the required dependencies:
+
+`pipenv --python 3.6.3 install`
+
+Set the `AIRFLOW_HOME` environment variable in the `.env` file to your absolute path:
```

Comment:
is this `workflow/airflow/.env`? Because it's in another directory and it's not used by `docker-compose.yml` :thinking: